src/main/kotlin/uk/gov/justice/digital/hmpps/nomisprisonerapi/jpa/OffenderProgramProfile.kt

```diff
@@ -57,7 +57,7 @@ data class OffenderProgramProfile(
 
   @ManyToOne(fetch = FetchType.LAZY)
   @JoinColumn(name = "CRS_ACTY_ID")
-  val courseActivity: CourseActivity? = null,
+  val courseActivity: CourseActivity,
```

Comment:
One of those nullable columns in NOMIS that is never null - saves me `!!`ing all unnecessarily.